python/tests/unittests/test_sample.py

```diff
@@ -339,7 +339,7 @@ def test_sample_result_observe(shots_count):
         sample_result = observe_result.counts()
         sample_result.dump()
         # Should just have 3 measurement registers, one for each spin term.
-        want_register_names = ["I0I1Z2", "I0Z1I2", "Z0I1I2"]
+        want_register_names = ["IIZ", "IZI", "ZII"]
```

Comment:
I'm curious how this works with the term numbers dropped from the strings? Is it just because the register name drops those values?

Reply:
I agree with #68 and prefer the qiskit format for this.

Reply:
Ah I hadn't seen that, all good with me was just curious 👍